fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/model/RowType.java

```diff
@@ -29,22 +29,27 @@ public void apply(IDatabaseAdapter target) {
         target.createRowType(getSchemaName(), getObjectName(), columns);
     }
 
+    @Override
+    public void apply(Integer priorVersion, IDatabaseAdapter target) {
+        target.createRowType(getSchemaName(), getObjectName(), columns);
```

Comment:
similar comment to RowType.  we don't actually use it right now.

Reply:
same resolution as RowArrayType